src/wrapper/jnienv.rs

```diff
@@ -636,64 +638,64 @@ impl<'a> JNIEnv<'a> {
             JavaType::Method(_) => unimplemented!(),
             JavaType::Primitive(p) => {
                 let v: JValue = match p {
-                    Primitive::Boolean => (jni_unchecked!(
+                    Primitive::Boolean => (jni_non_void_call!(
```

Comment:
Just wondering if a macro expansion affects the code size significantly — does it make sense to add a single `exception_check` below?

Reply:
I don't think it makes sense changing it back to the `jni_unchecked` with an additional line for exceptions checking, `jni_non_void_call` does exactly what we need and handles the return value "for free".